docs/user/pings/index.md

```diff
@@ -5,22 +5,22 @@ Every Glean ping is in JSON format and contains one or more of the [common secti
 If data collection is enabled, the Glean SDK provides a set of built-in pings that are assembled out of the box without any developer intervention.  The following is a list of these built-in pings:
 
 - [`baseline` ping](baseline.md)
-- [`events` ping](events.md)
 - [`metrics` ping](metrics.md)
+- [`events` ping](events.md)
```

Comment:
Why does this need moving?

Reply:
To have the same order as it does in the table of contents (sidebar). (I mention this in the commit so that we also know it when we ever come back to it)